tools/topology/topology2/production/tplg-targets-cavs25.cmake

```diff
@@ -24,10 +24,13 @@ SDW_SPK_STREAM=SDW2-Playback,SDW_DMIC_STREAM=SDW0-Capture,SDW_AMP_FEEDBACK=false
 DMIC1_ID=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,SDW_AMP_FEEDBACK=false,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-tgl-rt711-rt1308-4ch.bin"
 
-"cavs-sdw\;sof-tgl-cs42l43-l3-cs35l56-l01\;PLATFORM=tgl,NUM_SDW_AMP_LINKS=2,SDW_DMIC=1,\
-SDW_SPK_STREAM=Playback-SmartAmp,SDW_SPK_IN_STREAM=Capture-SmartAmp,\
-SDW_DMIC_STREAM=Capture-SmartMic,SDW_JACK_OUT_STREAM=Playback-SimpleJack,\
-SDW_JACK_IN_STREAM=Capture-SimpleJack,SDW_AMP_FMT_24=true,SDW_JACK_FMT_24=true"
+"cavs-sdw\;sof-tgl-cs42l43-l3-cs35l56-l01\;NUM_SDW_AMP_LINKS=2,SDW_DMIC=1,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
+SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
```

Comment:
Why are we dropping the 24bit stuff here? I think we should probably add that on the other entries too, cs42l43 doesn't support 32-bits over the soundwire (not tried it, perhaps the extra bits just get ignored, but officially the hardware doesn't support) and also you will run out of soundwire bandwidth in a lot of configurations if sending 32-bit.

Reply:
> Why are we dropping the 24bit stuff here? I think we should probably add that on the other entries too, cs42l43 doesn't support 32-bits over the soundwire (not tried it, perhaps the extra bits just get ignored, but officially the hardware doesn't support) and also you will run out of soundwire bandwidth in a lot of configurations if sending 32-bit.

After https://github.com/thesofproject/sof/pull/9282 was merged, 24 bit is the default format for SoundWire. We don't need to set SDW_AMP_FMT_24 and SDW_JACK_FMT_24 anymore.